src/convolution.cpp

```diff
@@ -290,7 +290,7 @@ ConvolutionDescriptor::GetForwardOutputTensorWithLayout(const TensorDescriptor&
         }
     }
 
-    std::size_t out_c;
+    std::size_t out_c = wei_k;
```

Comment:
same as https://github.com/ROCmSoftwarePlatform/MIOpen/pull/2166/files#r1248135803
Hopefully we have better way to workaround this issue? @atamazov